test/test_backend/db/test_refresh_dataset_group_tables.py

```diff
@@ -0,0 +1,59 @@
+"""Tests
+
+How to run:
+    python -m unittest discover
+
+todo's
+ - Improve test input files by picking out some concept IDs from concept.csv and then filtering rows from other files
+"""
+import os
+import sys
+import unittest
+from pathlib import Path
+
+from backend.db.refresh_dataset_group_tables import refresh_dataset_group_tables
+
+THIS_DIR = Path(os.path.dirname(__file__))
+THIS_INPUT_DIR = THIS_DIR / 'input' / 'test_refresh_dataset_group_tables'
+VOC_INPUT_DIR = THIS_INPUT_DIR / 'test_vocab'
+PROJECT_ROOT = THIS_DIR.parent.parent.parent
+DOWNLOAD_DIR = PROJECT_ROOT / 'termhub-csets' / 'datasets' / 'prepped_files'
+sys.path.insert(0, str(PROJECT_ROOT))
+TEST_SCHEMA = 'test_n3c'
+
+
+# todo: could create some more test cases (i) too many expressions, (ii) >100k members, (iii) 0 members?
+class TestRefreshDatasetGroupTables(unittest.TestCase):
```

Comment:
## Automated tests
@Sigfried I created this because I have no idea why the indexes are not being created when refreshing vocab / counts tables. This is a good practice anyway, though.